Jint.Tests/Runtime/Debugger/BreakPointTests.cs

```diff
@@ -138,11 +138,11 @@ public void BreakpointOverridesStepOut()
 
             var engine = new Engine(options => options.DebugMode());
 
-            engine.BreakPoints.Add(new BreakPoint(4, 0));
-            engine.BreakPoints.Add(new BreakPoint(6, 0));
+            engine.DebugHandler.BreakPoints.Add(new BreakPoint(4, 0));
```

Comment:
`DebugHandler.AddBreakPoint` and `DebugHandler.RemoveBreakPoint` might offer cleaner encapsulation, the list could probably be private and only offer IReadOnlyList maybe?